<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_client</class>
 <widget class="QWidget" name="interface_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>969</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Style pour les QPushButton */
QPushButton {
    background-color: #4CAF50;  /* Vert */
    color: white;
    border: 2px solid #4CAF50;
    border-radius: 10px;
    padding: 10px;
	font-weight: bold;
	font-size: 15px
}

QPushButton:hover {
    background-color: #45a049;  /* Vert plus foncé */
}

QPushButton:pressed {
    background-color: #388E3C;  /* Vert encore plus foncé */
}

/* Style pour les QLineEdit */
QLineEdit {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

QComboBox{
	background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QDateEdit */
QDateEdit {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QSpinBox */
QSpinBox {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QDoubleSpinBox */
QDoubleSpinBox {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour le QTableView */
QTableView {
    background-color: white;
    gridline-color: #4CAF50;
    border: 1px solid #4CAF50;
	color:black;

}
QTableView:pressed {
	bacground-color:rgb(188, 188, 188);
	 gridline-color: #4CAF50;
    border: 1px solid #4CAF50;
	color:black;
}


QHeaderView::section {
    background-color: #4CAF50;
    color: white;
    padding: 4px;
    border: 1px solid #4CAF50;
}

/* Style pour les QLabel */
QLabel {
    color:black;
    font-weight: bold;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Effectuer une vente</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Deconnexion</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">background-color:green;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;Enregistrement des clients&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nom </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Nom_client_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>contact</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Contact_client_lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Adresse électronique</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="Adresse_electronique_client_lineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="Adresse_client_lineEdit"/>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Enregistrer_pushButton">
         <property name="text">
          <string>Enregistrer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Modifier_pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: blue;</string>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AnnulerpushButton">
         <property name="styleSheet">
          <string notr="true">background-color: white;
color: green;</string>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Liste des clients&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QFormLayout" name="formLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="code_client">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Rechercher_pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: green;</string>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Suprimer_pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: red;
color: white;</string>
       </property>
       <property name="text">
        <string>Suprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Imprimer_pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: white;
color: green;</string>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
